src/main/java/org/apache/hadoop/hdfs/server/namenode/NNAConstants.java

```diff
@@ -250,7 +250,7 @@
     reloadNamespace,
     log,
     history,
-    suggestions,
+    status,
```

Comment:
Hmm -- bit of a controversial change here. While I do want to do this -- we are already relying on `/suggestions`. We may want to revert this part back and keep the `suggestions` endpoint for now, but keep the `status.html` change.

In the future we can work on making endpoints better and deprecating bad ones.

Reply:
The Suggestions page is meant to highlight what needs work on the cluster in terms of data management. The issue is today it really operates more as a Status page -- but if you note the little "[Explain]" buttons it is intended for it to actually provide advice on what to do and areas to focus. The goal is for there to eventually be a button for something like "Put into a HAR" or "Just delete them" as well.

Regarding the capacity -- there is an impact on us if that endpoint changes. Feel free to reach out offline about details. Good practice also dictates that API should undergo deprecations before being changed. I am for the change; just at a later time. It will simplify your changes for now to not change it too FWIW.